var/spack/repos/builtin/packages/pmdk/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+import spack.architecture
+
+
+class Pmdk(Package):
+    """The Persistent Memory Development Kit (PMDK), formerly known as NVML,
+    is a library for using memory-mapped persistence, optimized specifically
+    for persistent memory
+    """
+
+    homepage = "http://pmem.io/pmdk/"
+    url      = "https://github.com/pmem/pmdk/archive/1.5.tar.gz"
+    git      = "https://github.com/pmem/pmdk.git"
+
+    version('develop', branch='master')
+    version('1.5',     sha256='6b069d7207febeb62440e89245e8b18fcdf40b6170d2ec2ef33c252ed16db2d4')
+    # in the nvml -> pmdk name change, these hashes needed to be updated
```

Comment:
I'm not sure what you mean by updating these hashes, was there a prior `nvml` Spack package? If not, I don't think this comment is necessary.

Reply:
Ok, got it.  that's my comment.  It doesn't help anyone in 2019, so yeah, strike it